server/__tests__/products_test.js

```diff
@@ -5,20 +5,30 @@ describe('GET /api/products', () => {
 	test('responds with json and 200 status', function (done) {
 		request(app)
 			.get('/api/products/')
-			// .expect('Content-Type', /json/)
+			.expect('Content-Type', /json/)
 			.expect(200, done);
 	});
 })
+
 describe('PUT /api/products', () => {
-	test('accepts id and shop_quantity and responds with 200 status', function (done) {
+	test('responds with 200 status', function (done) {
```

Comment:
Think the test description could provide more context. For example if it was "when product exists, it updates the quantity"

Reply:
I agree. I'll amend the test names to be more descriptive